marketplace/dev/Dockerfile

```diff
@@ -1,22 +1,28 @@
-FROM gcr.io/cloud-builders/gcloud
+FROM python:3.7
```

Comment:
Can you use one from Marketplace or Google instead of this? Gives some assurance around OSPO compliance..